hazelcast/src/main/java/com/hazelcast/map/impl/LazyMapEntry.java

```diff
@@ -142,5 +150,16 @@ public int hashCode() {
     public String toString() {
         return getKey() + "=" + getValue();
     }
+
+    private void readObject(java.io.ObjectInputStream s) throws IOException, ClassNotFoundException {
```

Comment:
What about calling `defaultReadObject` method here? According to Effective Java it is a recommended practice. http://allenlsy.com/NOTES-of-Effective-Java-10/